package.json

```diff
@@ -939,6 +939,7 @@
         "azure-arm-storage": "^4.0.0",
         "azure-arm-website": "^5.3.0",
         "azure-storage": "^2.10.3",
+        "clipboardy": "^2.0.0",
```

Comment:
Since this is being used only in the test suites, this should be in the devDependecies